lnclient/ldk/ldk_logger.go

```diff
@@ -46,3 +53,38 @@ func mapLogLevel(logLevel ldk_node.LogLevel) logrus.Level {
 	logger.Logger.WithField("log_level", logLevel).Error("Unknown LDK log level")
 	return logrus.ErrorLevel
 }
+
+func InitLogger(ldkLogLevel string) {
+	LDKLogger = logrus.New()
+	LDKLogger.SetFormatter(&logrus.JSONFormatter{})
+	LDKLogger.SetOutput(os.Stdout)
+	ldkLogrusLogLevel, err := strconv.Atoi(ldkLogLevel)
+	if err != nil {
+		ldkLogrusLogLevel = int(logrus.InfoLevel)
+	}
+	LDKLogger.SetLevel(logrus.Level(ldkLogrusLogLevel))
+
+	if int(mapLogLevel(ldk_node.LogLevel(ldkLogrusLogLevel))) >= int(logrus.DebugLevel) {
```

Comment:
I don't think this is needed because the logs are only forwarded from LDK (which has its own line numbers)